docs/4-user-guide/2-zarf-packages/3-zarf-init-package.md

```diff
@@ -0,0 +1,142 @@
+
+Upon deployment, the init package creates a `zarf` namespace within your K8s cluster and deploys pods, services, and secrets to that namespace based on the components selected for deployment.
+
+## Required Component
+
+Zarf's mutation capabilities require that the [`zarf-agent`](../../9-faq.md#what-is-the-zarf-agent) component of the init package is deployed and active within the cluster, meaning that it cannot be disabled and is always running. This component intercepts requests to create resources and uses the `zarf-state` secret to mutate them to point to their air gap equivalents. It is automatically deployed whenever a `zarf init` command is executed.
+
+| Component               | Description                                                                                                                                           |
+| ----------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| zarf-agent              | A Kubernetes mutating webhook installed during `zarf init` that converts Pod specs and Flux GitRepository objects to match their air-gap equivalents. |
+
+:::note
+
+The `zarf-agent` will mutate any resources it sees that have not already been patched and don't have the `zarf.dev/agent: ignore` label applied. This label is automatically applied to all namespaces that exist prior to `zarf init`, and will prevent the `zarf-agent` from mutating system resources.  You can manually apply this label to any additional namespaces or resources after the fact to prevent the `zarf-agent` from acting upon them.  See the FAQ entry to learn more about [what happens to resources that were deployed prior to `zarf init`](../../9-faq.md#what-happens-to-resources-that-exist-in-the-cluster-before-zarf-init).
+
+:::
+
+## Core Components
+
+In addition to the required `zarf-agent` component, Zarf also offers components that provide additional functionality and can be enabled as needed based on your desired end-state.
+
+In most scenarios, Zarf will also deploy an internal registry using the three components described below. However, Zarf can be configured to use an already existing registry with the `--registry-*` flags when running `zarf init` (detailed information on all `zarf init` command flags can be found in the [zarf init CLI](../1-the-zarf-cli/100-cli-commands/zarf_init.md) section). This option skips the injector and seed process, and will not deploy a registry inside of the cluster. Instead, it uploads any images to the externally configured registry.
```

Comment:
should prob throw a note that you __MUST__ either deploy the internal registry, __OR__ configure init to use an external registry.